static/js/popovers.js

```diff
@@ -227,7 +227,24 @@ function do_set_reminder(msgid, timestamp) {
     reminder_message.reply_to = recipient;
     reminder_message.private_message_recipient = recipient;
     reminder_message.to_user_ids = people.email_list_to_user_ids_string(emails);
-    compose.schedule_message(reminder_message);
+
+    var row = $("[zid='" + msgid + "']");
```

Comment:
There should be a helper function in rows.js or somewhere similar that does this.